{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/pulse\/remotelogger\/connectionerror\/helpanchor"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/helpAnchor","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.helpAnchor"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"modules":[{"name":"Pulse","relatedModules":["Foundation"]}],"role":"symbol","title":"helpAnchor","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:10Foundation14LocalizedErrorPAAE10helpAnchorSSSgvp::SYNTHESIZED::s:5Pulse12RemoteLoggerC15ConnectionErrorO","extendedModule":"Foundation","platforms":[{"beta":false,"unavailable":false,"name":"iOS","introducedAt":"8.0","deprecated":false},{"beta":false,"unavailable":false,"name":"macOS","introducedAt":"10.10","deprecated":false},{"beta":false,"unavailable":false,"name":"tvOS","introducedAt":"9.0","deprecated":false},{"beta":false,"unavailable":false,"name":"watchOS","introducedAt":"2.0","deprecated":false}]},"hierarchy":{"paths":[["doc:\/\/com.github.pulse\/documentation\/Pulse","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/LocalizedError-Implementations"]]},"references":{"doc://com.github.pulse/documentation/Pulse/RemoteLogger":{"role":"symbol","title":"RemoteLogger","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"RemoteLogger"}],"abstract":[{"type":"text","text":"Connects to the remote server and sends logs remotely. In the current version,"},{"type":"text","text":" "},{"type":"text","text":"a server is a Pulse Pro app for macOS)."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"RemoteLogger"}],"url":"\/documentation\/pulse\/remotelogger"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/pulse\/remotelogger\/connectionerror\/localizederror-implementations"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError":{"role":"symbol","title":"RemoteLogger.ConnectionError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConnectionError"}],"abstract":[],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectionError"}],"url":"\/documentation\/pulse\/remotelogger\/connectionerror"},"doc://com.github.pulse/documentation/Pulse":{"role":"collection","title":"Pulse","abstract":[{"type":"text","text":"Logger and network inspector for Apple platforms."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse","kind":"symbol","type":"topic","url":"\/documentation\/pulse"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError/helpAnchor":{"role":"symbol","title":"helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/helpAnchor","kind":"symbol","type":"topic","url":"\/documentation\/pulse\/remotelogger\/connectionerror\/helpanchor"}}}